app/Providers/HorizonServiceProvider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\Facades\Gate;
+use Laravel\Horizon\Horizon;
+use Laravel\Horizon\HorizonApplicationServiceProvider;
+
+class HorizonServiceProvider extends HorizonApplicationServiceProvider
+{
+    /**
+     * Bootstrap any application services.
+     */
+    public function boot(): void
+    {
+        parent::boot();
+
+        // Horizon::routeSmsNotificationsTo('15556667777');
+        // Horizon::routeMailNotificationsTo('example@example.com');
+        // Horizon::routeSlackNotificationsTo('slack-webhook-url', '#channel');
+    }
+
+    /**
+     * Register the Horizon gate.
+     *
+     * This gate determines who can access Horizon in non-local environments.
```

Comment:
What is a non-local environment? I notice that we can see the shiny UI at http://api.wbaas.localhost/horizon/ in the local environment. I'm kinda assuming we don't want this in production but I'm unclear if it will be present there.

I don't understand how horizon or laravel would know minikube is local but GKE isn't. I'm imagining they look very similar from the perspective of the pod

Reply:
Laravel sets an `environment` which for us is either `local`, `staging` or `production`. If I understand this correctly, we should serve the Dashboard in all envs, but there is no way to authenticate when deployed to staging or production.